streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -4248,28 +4215,32 @@ public boolean maybePunctuateStreamTime() {
 
     @Test
     public void shouldPunctuateActiveTasks() {
```

Comment:
this test seems to be testing what it should. question is just whether it can be simplified (see below)

Reply:
Let's omit it then